README.md

```diff
@@ -1,117 +1,121 @@
-   If your analyzer greedy creates a connection to DataServer before one was actually started, you might want to disable fail-fast mode. This way the RPCs are queued until the chanel ready. Here is an [example](https://github.com/src-d/lookout-gometalint-analyzer/blob/7b4b37fb3109299516fbb43017934d131784f49f/cmd/gometalint-analyzer/main.go#L66).
-  - client/server RFC 3986 URI scheme support:
-    - go: use `pb.ToGoGrpcAddress` and `pb.Listen`
-    - python: use `lookout.sdk.grpc.to_grpc_address`
-
-Release Process
-=================
- - Make sure the code is up to date using `make protogen`.
- - Update `VERSION` in `python/setup.py` with the same version that you will use for the tag (manual step required until [#2](https://github.com/src-d/lookout-sdk/issues/2) is implemented).
- - Create the release tag.
-
- # License
-[Apache License v2.0](./LICENSE)
+<p align="center">
+  <a href="https://github.com/src-d/lookout-sdk/releases">
+    <img src="https://badge.fury.io/gh/src-d%2Flookout-sdk.svg"
+         alt="GitHub version">
+  </a>
+  <a href="https://pypi.org/project/lookout-sdk/">
+    <img src="https://badge.fury.io/py/lookout-sdk.svg"
+         alt="PyPI version">
+  </a>
+  <a href="https://travis-ci.org/src-d/lookout-sdk">
+    <img src="https://travis-ci.org/src-d/lookout-sdk.svg?branch=master"
+         alt="Build Status">
+  </a>
+  <a href="https://godoc.org/gopkg.in/src-d/lookout-sdk.v0/pb">
+    <img src="https://godoc.org/gopkg.in/src-d/lookout-sdk.v0?status.svg"
+         alt="GoDoc">
+  </a>
+  <a href="https://docs.google.com/document/d/1pqz-_SHO5BsJE-aa8o_bAY3r5vR67amnWN8-qZc2UgY">
+    <img src="https://img.shields.io/badge/source%7Bd%7D-design%20document-blue.svg"
+         alt="source{d} design document">
+  </a>
+</p>
+
+<p align="center"><b>
+    <a href="https://www.sourced.tech">Website</a> •
+    <a href="https://docs.sourced.tech">Documentation</a> •
+    <a href="https://blog.sourced.tech">Blog</a> •
+    <a href="http://bit.ly/src-d-community">Slack</a> •
+    <a href="https://twitter.com/sourcedtech">Twitter</a>
+</b></p>
+
+**Table of Contents**
+
+<!--ts-->
+   * [What does the SDK provide?](#what-does-the-sdk-provide)
+   * [Caveats](#caveats)
+      * [DataService](#dataservice)
+   * [Contributing](#contributing)
+      * [Community](#community)
+   * [Code of Conduct](#code-of-conduct)
+   * [License](#license)
+
+<!-- Added by: david, at: 2018-12-07T15:43+01:00 -->
+
+<!--te-->
+
+
+# What does the SDK provide?
+
+_For the complete documentation of **source{d} Lookout**, please take a look into [http://docs.sourced.tech/lookout](http://docs.sourced.tech/lookout)._
+
+_For detailed information about the different parts of Lookout, and how they interact you can go to the [lookout architecture guide](https://github.com/src-d/lookout/blob/master/docs/architecture.md)._
+
+**lookout-sdk** provides:
+
+- **proto [definitions](./proto)**,
```

Comment:
I'd remove the trailing `,`, it looks strange to me

Reply:
I also noticed it but I thought it's just me